packages/algob/test/builtin-tasks/compile.ts

```diff
@@ -58,6 +59,15 @@ describe("Compile task", () => {
   const f4 = "gold-asa.teal";
   const fhash = 2374470440; // murmur3 hash for f1 file
 
+  before(function () {
+    var isRoot = process.getuid && process.getuid() === 0;
+    if (isRoot) {
+      const warnMsg = chalk.yellowBright(`*** If running tests as root, make sure to have pyteal exist in sudo env (or install pyteal as root).
```

Comment:
how about renaming pyteal exist to pyteal installed ?

Reply:
ok